packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -31,15 +32,21 @@ export type RootScreenParamList = {
 
 const Stack = createNativeStackNavigator()
 
+type RootScreenProps = {
+  isPendingMandatoryCodePushUpdate?: boolean
+}
+
 /**
  * The top level navigator. Switches between sign on screens and main tab navigator
  * based on if the user is authed
  */
-export const RootScreen = () => {
+export const RootScreen = ({
+  isPendingMandatoryCodePushUpdate
+}: RootScreenProps) => {
   const dispatch = useDispatch()
   const accountStatus = useSelector(getAccountStatus)
   const showHomeStack = useSelector(getHasCompletedAccount)
-  const { updateRequired } = useUpdateRequired()
+  const { updateRequired: appUpdateRequired } = useUpdateRequired()
```

Comment:
Just wondering if we can make codepush hook and put it here?

Reply:
interesting. i think we can yeah, i had this assumption that `sync` had to be called from root component, but I do not think it does (esp. given that Sentry is actually the root component)